src/rshim.c

```diff
@@ -2376,6 +2390,56 @@ int rshim_fifo_size(rshim_backend_t *bd, int chan, bool is_rx)
   return is_rx ? read_cnt(bd, chan) : write_cnt(bd, chan);
 }
 
+static int rshim_load_cfg(void)
+{
+  char rshim_name[32] = "", dev_name[64] = "";
+  char *buf = NULL;
+  size_t n = 0;
+  FILE *file;
+  int index;
+
+  file = fopen(rshim_cfg_file, "r");
+  if (!file)
+    return -ENOENT;
+
+  while (getline(&buf, &n, file) != -1) {
+    if (sscanf(buf, "%31s%63s", rshim_name, dev_name) != 2)
+      continue;
+
+    if (strncmp(rshim_name, "rshim", 5) && strcmp(rshim_name, "none"))
+      continue;
+
+    if (strncmp(dev_name, "usb-", 4) && strncmp(dev_name, "pcie-", 5) &&
+        strncmp(dev_name, "pcie-lf", 7))
```

Comment:
checking for "pcie-" should cover both "pcie-" and "pcie-lf" cases. so the last check isn't necessary

Reply:
good catch. done.